assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py

```diff
@@ -894,10 +883,6 @@ def finetune(args: Namespace):
     args.save_strategy = args.evaluation_strategy
     args.save_steps = args.eval_steps
 
-    args.model_metadata = update_acft_metadata(metadata=metadata,
-                                               finetuning_task=args.task_name,
```

Comment:
why is this revert needed? latest accelerator would expect the model_metadata to be present right?

Reply:
We need to release Finetuning as a service components which are on older environment as all benchmarking is done on that.

We will re revert post release. Shivam has anyways started a component release for Model catalog

Reply:
This is for FTaaS release. Will revert once the release is done. (The FTaaS components depend on env 28 and latest env is 30. We will upgrade FTaaS after Ignite as all our benchmarking is done on env 28)